packages/ndla-ui/src/CompetenceGoalTab/CompetenceItem.tsx

```diff
@@ -277,55 +293,74 @@ const Item = ({ goal, t }: any) => {
   );
 };
 
-export type CompetenceTypeProps = 'LK06' | 'LK20' | 'coreElement';
+export type CompetenceTypeProps = 'competenceGoals' | 'coreElement';
+export type CompetenceGoals = {
+  title?: string;
+  competenceGoals: any;
+};
+export type CoreElementItems = {
+  title?: string;
+  coreElementItems: any;
+};
```

Comment:
Slik koden vår er no så bruker vi samme funksjon for å gruppere koder på læreplan. Dersom vi kan bruke samme navn på underelementa gjør det at vi slipp å komplisere/duplisere den koden. Det har ingenting å sei for designet for det er uansett skille på typene.
```suggestion
export type CompetenceGoals = {
  title?: string;
  elements: any;
};
export type CoreElementItems = {
  title?: string;
  elements: any;
};
```

Reply:
Enig, pusher endring straks